apistar/backends/sqlalchemy.py

```diff
@@ -16,7 +16,7 @@ def build(cls, settings: Settings):
         metadata = config.get('METADATA')
 
         kwargs = {}
-        if url.startswith('postgresql:'):  # pragma: no cover
+        if url.startswith('postgresql'):  # pragma: no cover
```

Comment:
what if someone is using `mongodb`? then that would cause a bug, maybe explicitly listing if statements is better?